app/models/g5_authenticatable/user.rb

```diff
@@ -38,7 +38,7 @@ def update_roles_from_auth(auth_data)
     end
 
     def clients
-      G5Updatable::ClientPolicy::Scope.new(self, G5Updatable::Client).resolve
+      G5Updatable::ClientPolicy::Scope.new(self, G5Updatable::Client).clients_from_client_and_location_roles
```

Comment:
I think that an indirect role (location) should entitle the user to it's parent role (client) on all the cases, and not on selected ones. If there's a case for "just listing the direct client roles", then it makes sense to have an specialized method to do so, but not the other way around since it's a primary use case. I'll let @maeve be our judge :)